pages/history.tsx

```diff
@@ -0,0 +1,15 @@
+import styles from '../styles/MoodHistory.module.css'
```

Comment:
One thing that is a standard here across the React ecosystem is that component names are PascalCased and match the PascalCased file names. Generally, there will be 1 component to 1 file (there are always exceptions). It's a cleanliness thing that they probably won't take points from Griffindor for.

Reply:
I don't really understand. Should I rename 'history.tsx' to 'MoodHistory.tsx'? That would also change the page name, since it's more of a page and less a component.

Reply:
That's typical. Matching your file names and component names probably wont get you any kudos as a dev, but not doing so might irk your senior devs.

Reply:
With `next` pages, you always use a default export. The name of the component isn't particularly material in that case.

Reply:
The above comment is mostly relevant to imported components, not so much `next` pages.